src/InteractiveWindow/Editor/IInteractiveWindowOperations2.cs

```diff
@@ -8,5 +8,15 @@ public interface IInteractiveWindowOperations2 : IInteractiveWindowOperations
         /// Copies the current selection to the clipboard.
         /// </summary>
         void Copy();
+
+        /// <summary>
+        /// Delete Line; Delete all selected lines, or the current line if no selection.  
+        /// </summary>
+        void DeleteLine();
```

Comment:
As we discussed offline, there's no ```CopyLine``` since there's no ```Edit.LineCopy``` in VS (whereas ```Edit.LineDelete``` and ```Edit.LineCut``` do exist).